### DISCUSSION TITLE
How do I filter search results for PRs with deleted head respositories?

### DISCUSSION THREAD

**Discussion by aaronfranke** [community/community#23714] [New to GitHub]
<p>I am trying to filter search results for PRs with deleted head repositories such that it says “unknown repository” in the PR:</p>
<p><img src="https://user-images.githubusercontent.com/1646875/181078869-21bb04f2-5e18-4746-a376-0ba1a4551a37.png" alt="Screenshot from 2020-10-12 09-06-06" data-base62-sha1="x9Z9m6GgAEvvnSClHG6y4ItOOyc" width="243" height="39"></p>
<p>There doesn’t seem to be any information on this in the documentation:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-missing-metadata" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="243" height="39">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-by-missing-metadata" target="_blank" rel="noopener nofollow ugc">Searching issues and pull requests - GitHub Docs</a></h3>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/aaronfranke">@aaronfranke</a> so for the filter of search results, what you’re asking for isn’t quite possible. It would be amazing if you could submit your idea through our feedback form:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://support-assets.github.com/assets/favicon-3dcd2c977a91b2e3f396b300c5ff0d572295ca632ddb7ba1adcacc1ea68dd5c0.ico" class="site-icon" width="16" height="16">
      <a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">support.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://support.github.com/contact/feedback" target="_blank" rel="noopener">Share feedback - GitHub Support</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>However, I’m wondering if you can’t accomplish what you’re trying to do by adjusting your search itself, rather than filtering the results you get?</p>
<p>Using something even as basic as <code>is:pr "unknown repository"</code>  (along with other qualifiers) might be a decent start, maybe?</p>
<p>Let me know what you think, and if that doesn’t get you on the right path. Would be curious to know if I’m misunderstanding your intent.</p>
